runtime/queries/capnp/indents.scm

```diff
@@ -0,0 +1,34 @@
+[
+  (annotation_targets)
+  (const_list)
+  (enum)
+  (interface)
+  (implicit_generics)
+  (generics)
+  (group)
+  (method_parameters)
+  (named_return_types)
+  (struct)
+  (struct_shorthand)
+  (union)
+] @indent
+
+((struct_shorthand (property)) @aligned_indent
+  (#set! "delimiter" "()"))
+
+((const_list (const_value)) @aligned_indent
+  (#set! "delimiter" "[]"))
```

Comment:
I don't think we have `@aligned_indent`?

Reply:
Thanks @pickfire 
From https://docs.helix-editor.com/guides/indent.html#adding-indent-queries, no.
I replaced `@indent_end` by `@outer`, removed `@branch` and `@aligned_indent`.
Only "scope" property is supported from https://github.com/helix-editor/helix/blob/master/helix-core/src/indent.rs#L433